foundation-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -748,9 +851,65 @@ The Android SDK automatically registers an `Application.ActivityLifecycleCallbac
 * Tracking Deep Link click-through.
 * Tracking Push Message click-through
 * Tracking Local Notification click-through
+
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+
+**Objective-C**
+
+ This method should be called to support the following use cases:
+
+ * Tracking Deep Link click-throughs
+   * From `application:didFinishLaunchingWithOptions`
+   * Extract `userInfo` from `UIApplicationLaunchOptionsURLKey`
+ * Tracking Push Message click-through
```

Comment:
Push message also doesn't need to be capitalized - so it can just be "Tracking push message clickthrough".

This goes for all mentions of push messages.